WebService/appsettings.ini

```diff
@@ -168,41 +168,50 @@ twin_writes_per_second = 10
 # A value too high (e.g. > 10) can decrease the overall throughput due to context switching.
 # A value too low (e.g. < 2) can decrease the overall throughput due to the time required to
 # loop through all the devices, when dealing we several thousands of devices.
+# The default value is 4
 telemetry_threads = 4
 
 # While connecting all the devices, limit the number of connections waiting to be
 # established. A low number will slow down the time required to connect all the devices.
 # A number too high will increase the number of threads, overloading the CPU.
+# The default value is 200
 max_pending_connections = 200
 
 # While sending telemetry, limit the number of messages waiting to be delivered.
 # A low number will slow down the simulation.
 # A number too high will increase the number of threads, overloading the CPU.
+# The default value is 1000
 max_pending_telemetry_messages = 1000
 
 # While writing device twins, limit the number of pending operations waiting to be completed.
+# The default value is 50
 max_pending_twin_writes = 50
 
 # When simulating behavior for all the devices in a thread, slow down if the loop through
-# all the devices takes less than N msecs. This is also the minimum time between two
+# all the devices takes fewer than N msecs. This is also the minimum time between two
```

Comment:
isn't "less" the right word here? the sentence is about the time spent, e.g. "the operation takes less than 1 hour" vs "the operation takes fewer than 1 hour"

[same below]

Reply:
The rule I follow/learned is that if the item is countable (milliseconds, in this case) then you use "fewer". If the item is not countable ("less hot", "less water") then you'd go with "less". I think that your example of "less than one hour" sounds correct though. "Fewer than one hour" would sound odd, but "Fewer than two hours" would sound correct, to me at least.

Reply:
there's an interesting note here https://data.grammarbook.com/blog/definitions/fewer-v-less/ :

> the expression "less than" is used in front of a plural noun that denotes a measure of distance, amount, or time